### DISCUSSION TITLE
The antenna of the ESP32 on the AI Deck has fallen off.

### DISCUSSION THREAD

**Discussion by docckey** [bitcraze/discussions#1259] [Q&A]
The antenna of the ESP32 on the AI Deck has fallen off. Actually, I'm not quite sure if it's an antenna. Now, I can't find it. I would like to ask if there is any solution or under what circumstances it can be determined that the part will not affect the operation of the AI deck. I also want to know where the antenna is located on the deck.

![98834FC5E6BCAA4DA9855CDAAE641D59](https://github.com/bitcraze/discussions/assets/148048159/ee877cb8-277b-48ca-84fb-c7eb703c71ec)

**Comment by gemenerik** [comment]
Yes, this is missing the antenna. The antenna is used for the ESP32 connectivity. This implies that you e.g. cannot run the WiFi streaming example. See our [CPX system overview](https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/infrastructure/flashing/#system-architecture) for a nice graphic of what functionality broke (without the antenna basically the NINA WiFi block and its links get removed).

Answered how to replace in #916
